src/Ctct/Services/AccountService.php

```diff
@@ -36,18 +35,53 @@ public function getVerifiedEmailAddresses($accessToken, Array $params)
     }
 
     /**
+     * Create new verified email addresses. This will also prompt the account to send
+     * a verification email to the address.
+     * @param string $accessToken - Constant Contact OAuth2 Access Token
+     * @param array $emailAddresses - array of VerifiedEmailAddress to create
+     * @return array - array of VerifiedEmailAddress created
+     */
+    public function createVerifiedEmailAddresses($accessToken, Array $emailAddresses)
+    {
+        $baseUrl = Config::get('endpoints.base_url') . Config::get('endpoints.account_verified_addresses');
+
+        $url = $this->buildUrl($baseUrl);
+        $response = parent::getRestClient()->post($url, parent::getHeaders($accessToken), $emailAddresses);
```

Comment:
The array $emailAddresses is not converted to JSON before it is posted to the API. Currently there are no unit tests for this method, so the tests can pass while this does not work. In addition, this endpoint only accepts a single email address to be verified.